source/_includes/integrations/manual_config_steps.md

```diff
@@ -0,0 +1,31 @@
+{% capture name %}{{ include.name | default: page.title }}{% endcapture %}
```

Comment:
Unless we have more cases, we should not extract things.

At this point, there is only a single use case for this one, so we should not make it a generic include?